DomainContent/AvatarStore/attachmentItemScript.js

```diff
@@ -35,6 +35,20 @@
     var firstGrab = true;
 
     var attachDistance = ATTACH_DISTANCE;
+
+    var attachFunction = function(){
+        var posEye = MyAvatar.getEyePosition();
+        var posFoot = MyAvatar.getJointPosition("LeftFoot");
+
+        if (posEye == {x: 0, y:0, z:0} ||
+            posFoot == {x: 0, y:0, z:0} || !HMD.active){
```

Comment:
strictness will never work between 2 vectors, a better alternative to compare them is to use `Vec3.equal(posEye, Vec3.ZERO)`